### PR TITLE
Update admin dashboard to be more user friendly

### PR DESCRIPTION
**Pull request by florian-vuillemot**
image

### CODE REVIEW

**Comment by florian-vuillemot** [comment]
I didn't use the theme component (`shared/components/theme`), should I?

File: src/pages/admin/index.tsx [hunk 1]
```diff
@@ -1,91 +1,141 @@
...
-        },
-        {
-            title: 'Submissions management',
-            onClick: () => {
-                navigate('/admin/submissions')
-            },
-        },
-    ]
+const Title = styled.p`
+    color: ${(props) => props.color || '#000000'};
```
Comment (MarlonLH) [inline src/pages/admin/index.tsx:44]: Keep the same color for all the titles.

```suggestion
    color: #000000;
```

File: src/pages/admin/index.tsx [hunk 2]
```diff
@@ -1,91 +1,141 @@
...
+    font-size: 48px;
```
Comment (MarlonLH) [inline src/pages/admin/index.tsx:45]: Decrease the text size
```suggestion
    font-size: 38px;
```

File: src/pages/admin/index.tsx [hunk 3]
```diff
@@ -1,91 +1,141 @@
...
+
+const ContentWrapper = () => (
+    <TextContainer>
+        <TitleWrapper>
+            <Title color="#ff9900">Content</Title>
+        </TitleWrapper>
+        <Button
+            redirection={() => navigate('/admin/create')}
+            icon={AiFillFileAdd}
+            color="#ff9900"
```
Comment (MarlonLH) [inline src/pages/admin/index.tsx:64]: Keep it black

```suggestion
            color="#000000"
```

File: src/pages/admin/index.tsx [hunk 4]
```diff
@@ -1,91 +1,141 @@
...
+
+const SectionWrapper = () => (
+    <TextContainer>
+        <TitleWrapper>
+            <Title>Sections</Title>
+        </TitleWrapper>
+        <Button
+            redirection={() => navigate('/admin/sections/create')}
+            icon={AiFillFileAdd}
+            color="#ff9900"
```
Comment (MarlonLH) [inline src/pages/admin/index.tsx:84]: Keep it black

```suggestion
            color="#000000"
```

File: src/pages/admin/index.tsx [hunk 5]
```diff
@@ -1,91 +1,141 @@
...
+
+const LivestreamWrapper = () => (
+    <TextContainer>
+        <TitleWrapper>
+            <Title color="#ff9900">Livestream</Title>
+        </TitleWrapper>
+        <Button
+            redirection={() => navigate('/admin/livestream')}
+            icon={FcEnteringHeavenAlive}
+            color="#ff9900"
```
Comment (MarlonLH) [inline src/pages/admin/index.tsx:104]: Keep it black

```suggestion
            color="#000000"
```

File: src/pages/admin/index.tsx [hunk 6]
```diff
@@ -1,91 +1,141 @@
...
+
+const UserSubmissionWrapper = () => (
+    <TextContainer>
+        <TitleWrapper>
+            <Title>User submission</Title>
+        </TitleWrapper>
+        <Button
+            redirection={() => navigate('/admin/submissions')}
+            icon={FaUserFriends}
+            color="#ff9900"
```
Comment (MarlonLH) [inline src/pages/admin/index.tsx:118]: Keep it black

```suggestion
            color="#000000"
```

File: src/pages/admin/index.tsx [hunk 7]
```diff
@@ -1,91 +1,141 @@
...
+)
+
+const LivestreamWrapper = () => (
+    <TextContainer>
+        <TitleWrapper>
+            <Title color="#ff9900">Livestream</Title>
+        </TitleWrapper>
+        <Button
+            redirection={() => navigate('/admin/livestream')}
+            icon={FcEnteringHeavenAlive}
```
Comment (MarlonLH) [inline src/pages/admin/index.tsx:103]: Keep the same icon

```suggestion
            icon={SiGoogletagmanager}
```

File: src/pages/admin/index.tsx [hunk 8]
```diff
@@ -1,91 +1,141 @@
...
+)
+
+const UserSubmissionWrapper = () => (
+    <TextContainer>
+        <TitleWrapper>
+            <Title>User submission</Title>
+        </TitleWrapper>
+        <Button
+            redirection={() => navigate('/admin/submissions')}
+            icon={FaUserFriends}
```
Comment (MarlonLH) [inline src/pages/admin/index.tsx:117]: Keep the same icon

```suggestion
            icon={SiGoogletagmanager}
```

**Review by MarlonLH — CHANGES_REQUESTED** [review]
These components aren't designed for an admin panel, so they won't render well anyway.
Please keep a consistent color to prevent a high color complexity.

**Comment by florian-vuillemot** [comment]
image

**Review by MarlonLH — APPROVED** [review]

**Pull request merged**